test/server/graphql/v2/mutation/MemberMutations.test.ts

```diff
@@ -0,0 +1,268 @@
+          account: { id: idEncode(collective.id, IDENTIFIER_TYPES.ACCOUNT) },
+          description: 'role changed to ACCOUNTANT',
+          role: roles.ACCOUNTANT,
+          since: new Date('01 January 2022').toISOString(),
+        },
+        collectiveAdminUser,
+      );
+
+      const activity = await models.Activity.findAll({
+        where: {
+          type: ActivityTypes.COLLECTIVE_CORE_MEMBER_EDITED,
+          CollectiveId: collective.id,
+        },
+      });
+      const editedMember = result.data.editMember;
+
+      expect(result.errors).to.not.exist;
+      expect(activity.length).to.equal(1);
+      expect(activity[0].data.memberCollective.id).to.equal(collectiveMemberUser.id);
+      expect(editedMember.role).to.equal(roles.ACCOUNTANT);
+      expect(editedMember.description).to.equal('role changed to ACCOUNTANT');
+      expect(editedMember.since.toISOString()).to.equal(new Date('01 January 2022').toISOString());
+    });
+    it('must be authenticated as an admin of the collective', async () => {
+      const randomUser = await fakeUser();
+      const result = await utils.graphqlQueryV2(
+        editMemberMutation,
+        {
+          memberAccount: { id: idEncode(collectiveMemberUser.CollectiveId, IDENTIFIER_TYPES.ACCOUNT) },
+          account: { id: idEncode(collective.id, IDENTIFIER_TYPES.ACCOUNT) },
+          description: 'role changed to ACCOUNTANT by a random user',
+          role: roles.ACCOUNTANT,
+          since: new Date('01 January 2022').toISOString(),
+        },
+        randomUser,
+      );
+      expect(result.errors).to.have.length(1);
+      expect(result.errors[0].message).to.equal('Only admins can edit members.');
+    });
+    it('must not change the role of the last admin', async () => {
+      const result = await utils.graphqlQueryV2(
+        editMemberMutation,
+        {
+          memberAccount: { id: idEncode(collectiveAdminUser.CollectiveId, IDENTIFIER_TYPES.ACCOUNT) },
+          account: { id: idEncode(collective.id, IDENTIFIER_TYPES.ACCOUNT) },
+          description: 'collectiveAdminUser(last admin) role changed to ACCOUNTANT',
+          role: roles.ACCOUNTANT,
+          since: new Date('01 January 2022').toISOString(),
+        },
+        collectiveAdminUser,
+      );
+      expect(result.errors).to.have.length(1);
+      expect(result.errors[0].message).to.equal('There must be at least one admin for the account.');
+    });
+    it('can only update role to accountant, admin or member', async () => {
+      [roles.ADMIN, roles.MEMBER, roles.ACCOUNTANT, roles.HOST].forEach(async role => {
+        const result = await utils.graphqlQueryV2(
+          editMemberMutation,
+          {
+            memberAccount: { id: idEncode(collectiveMemberUser.CollectiveId, IDENTIFIER_TYPES.ACCOUNT) },
+            account: { id: idEncode(collective.id, IDENTIFIER_TYPES.ACCOUNT) },
+            description: `role changed to ${role}`,
+            role: roles.ADMIN,
```

Comment:
We intend to test whether we can change the role to `role` here

```suggestion
            role,
```